lib/src/main/kotlin/pl/allegro/logic/ClientLogicOptIn.kt

```diff
@@ -0,0 +1,11 @@
+package pl.allegro.logic
```

Comment:
this whole thing is not needed in library

Reply:
done